content/the-incoming-summer-of-clippy.md

```diff
@@ -0,0 +1,61 @@
++++
+path = "9999/12/31/the-incoming-summer-of-clippy"
+title = "The incoming Summer of Clippy"
+authors = ["blyxyas"]
+
+[extra]
+team = "the Clippy team"
+team_url = "https://www.rust-lang.org/governance/teams/dev-tools#team-clippy"
++++
+
+The Clippy project will be on feature-freeze for 12 weeks, starting from Rust
+1.89.0 beta (June 26th 2025) to September 18th 2025 (Rust 1.89.0 stable
+release). During this time no new features will be accepted, only bug fixes.
+
+This *summer of Clippy* comes from a lack of the necessary capacity needed
+to maintain all the current lints (about 750 of them 😱) and still add new ones.
+We need to care for the Clippy project the same way that Clippy cares about our
+code, and note that every single one of them needs maintaining and accounting
+for current and future changes in the compiler as a whole.
+
+As a team we’ll focus on making contributing the best it can be, and mark the
+issues that could be great as a starting point to get familiar with the project.
```

Comment:
I would move this to the `Can I help?` section.

Reply:
Imo, having this section in the introductory section gives to the reader some structure. Like:

What's happening -> Why it's happening -> What we'll be doing as a team. The other two sections are separate as they're meant to be read as a more involved user/contributor.